src/Notification/InlineNotification.svelte

```diff
@@ -81,10 +103,7 @@
       <NotificationButton
         {iconDescription}
         {notificationType}
-        on:click={() => {
-          open = false;
-          dispatch('close');
-        }} />
+        on:click={close()} />
```

Comment:
The `close` function is invoked here, causing the notification to unexpectedly close.

```diff
- on:click={close()} />
+ on:click={close} />
```

Reply:
Sorry my bad. I was rushing this a bit.